index.html

```diff
@@ -79,4 +77,4 @@ <h2>Made with ❤️️ by Horiseon</h2>
     </div>
```

Comment:
Also the last div here seems like it is on the bottom of a page and see what HTML semantic element can be used to replace the bottom `div`